package.json

```diff
@@ -3,31 +3,46 @@
   "version": "0.1.0",
   "description": "Utilities for working with Substrait plans in Javascript projects",
   "main": "dist/index.js",
+  "type": "module",
+  "bin": {
+    "substrait": "dist/index.js"
+  },
   "scripts": {
     "clean": "rimraf dist",
-    "build": "npm run clean && npm run make-dist && npm run transpile",
+    "build": "npm run clean && npm run clean-proto && npm run proto && npm run transpileProto && npm run make-dist && npm run transpile",
     "make-dist": "mkdirp dist",
-    "clean-proto": "rimraf src/generated && mkdirp src/generated",
-    "proto": "pbjs --target static-module --path substrait-spec/proto -o src/generated/definitions.js substrait/plan.proto",
+    "clean-proto": "rimraf src/generated && mkdirp src/generated && mkdirp dist/generated",
+    "proto": "pbjs -w scripts/wrapper.js --target static-module --path substrait-spec/proto -o dist/generated/definitions.js substrait/plan.proto",
     "transpile": "tsc",
-    "transpileProto": "pbts -o src/generated/definitions.d.ts src/generated/definitions.js",
+    "transpileProto": "pbts -o src/generated/definitions.d.ts dist/generated/definitions.js",
     "prepare": "npm run clean-proto && npm run proto && npm run transpileProto",
     "lint": "eslint '**/*.ts'",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "lint:fix": "npm run lint -- --fix",
+    "test": "$HOME/.nvm/nvm-exec jest"
   },
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
+    "@viz-js/viz": "^3.2.3",
+    "commander": "^11.0.0",
     "d3": "^7.8.5",
```

Comment:
Is d3 still used?  Can we remove this?

Reply:
yes, we can remove this. Now, substrait-JS has no d3 dependency. D3 will be required when we build the fiddle on top of this.